### PR TITLE
Serve bulk reports

### PR DESCRIPTION
**Pull request by olovy**
* Add `bulk:execution` with document counts and links to reports
* Serve bulk whelktool reports from housekeeping

Depends on https://github.com/libris/definitions/pull/504

### CODE REVIEW

File: whelktool/src/main/java/whelk/datatool/bulkchange/BulkJob.java [hunk 1]
```diff
@@ -59,14 +80,65 @@ public void run() {
             printScriptLogHeader(tool, jobDoc);
             logger.info(String.format("Running %s: %s", JOB_TYPE, id));
 
+            storeUpdate(doc -> doc.setStatus(Completed));
```
Comment (kwahlin) [inline whelktool/src/main/java/whelk/datatool/bulkchange/BulkJob.java:83]: Is this intentional?

**Review by kwahlin — APPROVED** [review]
Great job! :star: Looks good. Just a small thing that I suspect is an error, see comment.

File: whelktool/src/main/java/whelk/datatool/bulkchange/BulkJob.java (hunk 1 above)
Comment (olovy) [reply whelktool/src/main/java/whelk/datatool/bulkchange/BulkJob.java:83]: No!!

Comment (olovy) [inline whelktool/src/main/java/whelk/datatool/bulkchange/BulkJob.java:83]:
```suggestion
```

**Pull request merged**